Jenkinsfile_CNP

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "send-letter-service"
+def component = "api"
+
+def channel = '#platform-engineering'
+
+withPipeline(type , product, component) {
+
+    after('checkout') {
+        echo 'send-letter-service out'
```

Comment:
is this useful?

Reply:
@rkondratowicz Not much tbh.We can only enable slack notifications.